.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 16
+          node-version: 12
```

Comment:
Just wondering about this. I'm guessing the production server is running on this node version?

Reply:
I followed the Angular compatibility guide. Previously, it would fail at package building on Github due to the node version. It's not that important now, but it will be once we have functioning linting/unit tests setup on Github.